test/unit/models/inbound/authorizations.model.test.ts

```diff
@@ -53,7 +52,7 @@ describe('InboundAuthorizationModel', () => {
       counter: '1'
```

Comment:
maybe adding the type for `authenticationValue` would prevent as from casting `as any` later?